widgetastic_manageiq/__init__.py

```diff
@@ -4356,3 +4356,79 @@ def fill(self, value):
             return False
         self._all_item_elements[value or self.selected_item].click()
         return True
+
+
+class MigrationPlansList(Widget):
+    """Represents the list of MIgration Plans."""
+
+    ROOT = ParametrizedLocator('.//div[contains(@class,{@list_class|quote})]')
+    ITEM_LOCATOR = ParametrizedLocator('.//div[contains(@class,"{@list_class}__list-item")]')
+    # ITEM_TEXT_LOCATOR helps locate name of the Migration plan. ITEM_LOCATOR.text does not suffice.
+    # Also, ITEM_LOCATOR does not yield element which can be clicked to navigate to details page.
+    ITEM_TEXT_LOCATOR = './/div[contains(@class,"list-group-item-heading")]'
+    ITEM_TIMER_LOCATOR = './/div[./span[contains(@class,"fa-clock-o")]]'
+
+    def __init__(self, parent, list_class, logger=None):
+        Widget.__init__(self, parent, logger=logger)
+        self.list_class = list_class
+
+    @property
+    def all_items(self):
+        return [item.text for item in self.browser.elements(self.ITEM_TEXT_LOCATOR)]
+
+    def get_vm_count_in_plan(self, plan_name):
+        return self._get_plan_locator(plan_name).find_element_by_xpath(
+            './/div[./span[contains(@class,"pficon-virtual-machine") ' +
```

Comment:
Required: use .format() here

Reply:
No need for .format here. but I think "+" is not needed as well. Removing it.